doc/redirects.json

```diff
@@ -3833,9 +3833,6 @@
   "updating-plugins-in-nixpkgs": [
     "index.html#updating-plugins-in-nixpkgs"
   ],
-  "vim-out-of-tree-overlays": [
-    "index.html#vim-out-of-tree-overlays"
```

Comment:
Please don't just delete entries. Instead redirect old identifiers to some new or replacement item or to a release note in the Nixpkgs manual that explains why the entry is gone.

Reply:
I tried, it errored anyway. The page was not helpful tbh

```json
  "vim-out-of-tree-overlays": [
    "index.html#vim"
```
How to redirect it? This doesn't work

Reply:
Yes, sorry for the additional friction. :( There's [work planned until the next release](https://discourse.nixos.org/t/splitting-the-nixpkgs-and-nixos-manuals-into-pages/55466/10) to improve the DX on that front, step by step.

Since you remove the `vim-out-of-tree-overlays` identifier from the markdown source, that key must go away, but the location `index.html#vim-out-of-tree-overlays` must stay around so it can be converted into a redirect -- append it under the key that it's supposed to redirect to. If it's a new release notes entry, add an anchor to that entry and add the corresponding redirects key which should have two entries:
1. then entry's current location
2. `index.html#vim-out-of-tree-overlays`

Reply:
> Those redirects helper commands in the docs shell environment are supposed to do that source manipulation for you, and clearly need more work.

Yeah, I get `command not found` on them

Reply:
> I get `command not found` on them

You need `nix-shell ./doc/shell.nix` to get that command

Reply:
Ran `redirects remove-and-redirect vim-out-of-tree-overlays vim`, I hope this is enough